pgmpy/readwrite/XMLBIF.py

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env python
 
+import string
+import re
+from io import BytesIO
+
+# TODO input and output state
```

Comment:
@finnhacks42 Can you add in the support for input and output state, our CPD representation supports input and output state.